src/podping_hivewriter/constants.py

```diff
@@ -1,6 +1,6 @@
 LIVETEST_OPERATION_ID = "pplt"
 PODPING_OPERATION_ID = "pp"
-STARTUP_OPERATION_ID = "ppst"
+STARTUP_OPERATION_ID = "_startup"
```

Comment:
If this is intended to be a suffix to either `LIVETEST_OPERATION_ID` or `PODPING_OPERATION_ID` we should probably indicate as such.

Reply:
Better comment elsewhere.